src/test/java/au/com/southsky/jfreesane/SaneClientAuthenticationTest.java

```diff
@@ -16,16 +16,10 @@
 public class SaneClientAuthenticationTest {
 
   @Test
-  public void testSaneClientAuthentication() {
-    SaneClientAuthentication sca = new SaneClientAuthentication();
-    Assert.assertNotNull(sca);
-  }
-
-  @Test
   public void testSaneClientAuthenticationWithMissingFileDoesNotFail() {
     String filepath = "NON_EXISTENT_PATH_" + UUID.randomUUID().toString();
     SaneClientAuthentication sca = new SaneClientAuthentication(filepath);
-    Assert.assertNotNull(sca);
+    sca.getCredentialForResource(null);
```

Comment:
Imo that shouldn't matter because there's no check whether the path exists or not and thus an IOException because of an inexisting file is thrown. Let's try it out - what argument should I pass to `getCredentialsForResource`?